wremnants/datasets/datasetDict_v9.py

```diff
@@ -13,40 +13,48 @@
                                      "{BASE_PATH}/SingleMuon/NanoV9Run2016GDataPostVFP_{NANO_PROD_TAG}/*/*/*.root",
                                      "{BASE_PATH}/SingleMuon/NanoV9Run2016HDataPostVFP_{NANO_PROD_TAG}/*/*/*.root",
                                      ],
+                      'group': "Data",
+        
     },
     'ZmumuPostVFP' : { 
                    'filepaths' :
                     ["{BASE_PATH}/DYJetsToMuMu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MCPostVFP_{NANO_PROD_TAG}/*/*/*.root"],
                    'xsec' : xsec_ZmmPostVFP,
+                   'group': "Zmumu",
     },
     'ZtautauPostVFP' : { 
                    'filepaths' : 
                    ["{BASE_PATH}/DYJetsToTauTau_M-50_AtLeastOneEorMuDecay_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MCPostVFP_{NANO_PROD_TAG}/*/*/*.root"],
                    # At least one tau->e or mu decay, so everything that's not all other decays
                    'xsec' : xsec_ZmmPostVFP*Z_TAU_TO_LEP_RATIO,
+                   'group': "Ztautau",
     },
 
     'WplusmunuPostVFP' : { 
                       'filepaths' : 
                       ["{BASE_PATH}/WplusJetsToMuNu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MCPostVFP_{NANO_PROD_TAG}/*/*/*.root"],
-                      'xsec' : xsec_WpmunuPostVFP
+                      'xsec' : xsec_WpmunuPostVFP,
+                      'group': "Wmunu",
     },
     'WminusmunuPostVFP' : { 
                       'filepaths' : 
                       ["{BASE_PATH}/WminusJetsToMuNu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MCPostVFP_{NANO_PROD_TAG}/*/*/*.root"],
-                      'xsec' : xsec_WmmunuPostVFP
+                      'xsec' : xsec_WmmunuPostVFP,
+                      'group': "Wmunu",
     },
 
     'WplustaunuPostVFP' : { 
                          'filepaths' : 
                          ["{BASE_PATH}/WplusJetsToTauNu_TauToMu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MCPostVFP_{NANO_PROD_TAG}/*/*/*.root"],
                          'xsec' : BR_TAUToMU*xsec_WpmunuPostVFP,
+                         'group': "Wtaunu",
```

Comment:
I really don't like the need to manually keep the group specified here and the groups where it's actually placed in "datagroups" in sync. I suggest either we get rid of this and don't rely on it at all or we only build the groups in datagroups from this (I prefer the former, honestly)

Reply:
ok, we agreed to keep this as you already said. True that there is still some redundancy between this name and what becomes the key of the group dictionary later, albeit they don't have to match, since the "group" keyword here in datasetDict_v9 is just a handle to facilitate grouping of samples later in datagroups.py when the narf datasets are created